WordPress/gutenberg
# Hide parent selector in widget area

## Description

Fix #25908.

Introduced a new block meta data: `hideParentSelector`, to indicate if the block parent selector should show or not. Not sure if it's the optimal solution?

## How has this been tested?



1. Go to Widgets screen
2. Select any legacy widget
3. Hover over the parent selector in the block toolbar
4. Observe that the parent selector isn't showing

## Screenshots 
![image](https://user-images.githubusercontent.com/7753001/95706597-db0c4780-0c89-11eb-9b3c-f04f17be5b22.png)

## Types of changes




Bug fix

## Checklist:
